workflows/testing/test_workflow/manifest.py

```diff
@@ -0,0 +1,47 @@
+import yaml
+
+class BuildManifest:
+    @staticmethod
+    def from_file(path):
+        with open(path, 'r') as file:
+            hash = yaml.safe_load(file)
```

Comment:
hash? Not sure what this naming convention means?

Reply:
It means the author of this code comes from a Ruby background. What's the convention in Python?

Reply:
ohhh... I have no idea - hash to me means md5 or sha1...  Blob or data come to mind, maybe whatever yaml is called, if it was json i'd call it json.

Reply:
Should just be `data`, e.g. https://github.com/opensearch-project/opensearch-build/blob/c9885c0f7fe4fb6808cf3b34364ef91efcdeb825/tools/bundle-build/lib/manifest.py

Reply:
Done